crypto/src/proofs/transparent.rs

```diff
@@ -587,6 +565,217 @@ impl TryFrom<transparent_proofs::SwapProof> for SwapProof {
     }
 }
 
+/// Transparent proof for swap creation.
+///
+/// Swaps create an output NFT encoding the swap data so they are most similar to Output operations.
+///
+/// This structure keeps track of the auxiliary (private) inputs.
+#[derive(Clone, Debug)]
+pub struct SwapProof {
+    // The diversified base for the destination address.
+    pub g_d: decaf377::Element,
+    // The transmission key for the destination address.
+    pub pk_d: ka::Public,
+    // The value of asset 1 in the swap.
+    pub value_t1: Value,
+    // The value of asset 2 in the swap.
+    pub value_t2: Value,
+    // The fee value associated with the swap.
+    // TODO: should always be of penumbra token type, maybe this should
+    // be uint64 or a hardcoded value?
+    pub value_fee: Value,
```

Comment:
Yeah, since the asset ID is fixed, this should just be a `u64`.  I think using `_amount` is a good convention for amounts, so this could be `fee_amount: u64`.